tests/autogen_op/ALL/test_ALL_aten__native_batch_norm_legit_no_training_default.py

```diff
@@ -11,8 +11,13 @@ class AtenModule(torch.nn.Module):
     def __init__(self):
         super().__init__()
 
-    def forward(self, *args, **kwargs):
-        return torch.ops.aten._native_batch_norm_legit_no_training.default(*args, **kwargs)
+    def forward(self, input, weight, bias, running_mean, running_var, momentum, eps):
+        # Make sure the var >= 0
+        running_var = running_var**2
```

Comment:
> # Make sure the var >= 0

No guarantee it is not `0 `

Reply:
Exact 0 is mathematically valid as variance is the expected value of a square.  I'd say squaring a random vector (instead of taking its absolute value) is ingenious.

Var(*X*) = E[(*X* &minus; E[*X*])<sup>2</sup>]

Reply:
Agreed. Thank you